s3torchconnectorclient/python/tst/unit/test_mountpoint_s3_client.py

```diff
@@ -25,6 +25,9 @@
 REGION = "us-east-1"
 MOCK_BUCKET = "mock-bucket"
 ENDPOINT = "https://s3.us-east-1.amazonaws.com"
+EMPTY_ENDPOINT = ""
+NONE_ENDPOINT = None
+INVALID_ENDPOINT = "https://us-east-1.amazonaws.com"
```

Comment:
Thanks for adding these tests. Can we cover these scenarios with a parameterized test such as in https://github.com/awslabs/s3-connector-for-pytorch/blob/main/s3torchconnector/tst/unit/test_s3mapdataset.py#L30-L37 ?
Can we also add another one with a completely invalid URI (say `"[INVALID]"`) with associated `pytest.raises` block? (Example https://github.com/awslabs/s3-connector-for-pytorch/blob/main/s3torchconnectorclient/python/tst/unit/test_mountpoint_s3_client.py#L241-L243)

Reply:
I made the changes. For invalid URI case, I need to use the put_object method on MountpointS3Client as the URI validation occurs when trying to connect.